deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobTest.java

```diff
@@ -441,6 +450,52 @@ public void resumeIngestWorkObjectTest() throws Exception {
         verify(jobStatusFactory).setTotalCompletion(eq(jobUUID), eq(3));
     }
 
+    @Test
+    public void pauseAndResumeTest() throws Exception {
+        PID workPid = makePid(RepositoryPathConstants.CONTENT_BASE);
+        WorkObject work = mock(WorkObject.class);
+        Bag workBag = setupWork(workPid, work);
+
+        String mainLoc = "pdf.pdf";
+        String mainMime = "application/pdf";
+        PID mainPid = addFileObject(workBag, mainLoc, mainMime);
+
+        workBag.addProperty(Cdr.primaryObject, model.getResource(mainPid.getRepositoryPath()));
+
+        job.closeModel();
+
+        when(work.addDataFile(any(PID.class), any(URI.class),
+                anyString(), anyString(), anyString(), anyString(), any(Model.class)))
+                .thenReturn(mockFileObj);
+        when(mockFileObj.getPid()).thenReturn(mainPid);
+
+        when(depositStatusFactory.getState(depositUUID))
+                .thenReturn(DepositState.running)
+                .thenReturn(DepositState.running)
```

Comment:
Does DepositState.running need to be here twice?

Reply:
when multiple `thenReturn`s are chained like this, it means the first and second calls of `getState` will return `running`, and then all calls after that will return `paused`. So the number of runnings depended on the number of calls expected